classification/classification_network.py

```diff
@@ -99,63 +75,61 @@ def createEncoder(newEncoders):
     }
   """
   encoder = MultiEncoder()
-  encoder.addMultipleEncoders(newEncoders)
+  encoder.addMultipleEncoders(encoders)
   return encoder
 
 
 
-def createSensorRegion(network, sensorType, encoders, dataSource, numCats):
+def _createSensorRegion(network,
+                        regionName,
+                        sensorType,
+                        sensorParams,
+                        dataSource,
+                        encoders):
   """
   Initializes the sensor region with an encoder and data source.
 
   @param network      (Network)
+  
+  @param regionName   (str) Name for this region
```

Comment:
Cleanup docstring format. However you want to style it is fine, just be consistent. Please make these changes throughout.